app/models/developer_programming_language.rb

```diff
@@ -1,2 +1,4 @@
 class DeveloperProgrammingLanguage < ApplicationRecord
+  belongs_to :developer
+  belongs_to :programming_language
```

Comment:
Add `inverse_of` in intermediate tables.
See below.
https://github.com/jp-ryuji/playbook/wiki/Self-Review-Check-List#models-including-queries

Reply:
The seem inverse_of don't support in this case? @jp-ryuji 

There are a few limitations to :inverse_of support:

They do not work with :through associations.
They do not work with :polymorphic associations.
They do not work with :as associations.

Reply:
@ntrung2006 
So you're sure that this case matches one of the limitations?
This one? `They do not work with :through associations.`.

Reply:
@ntrung2006 
Alright!
My understanding is as follows.
You're saying that `inverse_of` is not supported in this case, so with or without `inverse_of` doesn't make any difference. 

Can you prove that?
I expect you to write some code.
It can be on `rails console`.
This is not included in the exercise.

So take your time (you don't need to worry about the deadline).
:)

Reply:
Okk @jp-ryuji :)